File: java2csharp/java/Samsung/ProfessionalAudio.FloatingConotrller.WidgetsLibrary/NonAvailableActivityException.java
package com.samsung.android.sdk.professionalaudio.widgets;

class NonAvailableActivityException extends Exception {

    /**
     * 
     */
    private static final long serialVersionUID = -1452800299632702425L;
    private static final String MSG = "Activity non available. It has been cleared by GC";

    @Override
    public String getMessage() {
        return MSG;
    }

}

File: java2csharp/csharp/Samsung/ProfessionalAudio.FloatingConotrller.WidgetsLibrary/NonAvailableActivityException.cs
using System;

namespace com.samsung.android.sdk.professionalaudio.widgets
{

	internal class NonAvailableActivityException : Exception
	{

		/// 
		private const long serialVersionUID = -1452800299632702425L;
		private const string MSG = "Activity non available. It has been cleared by GC";

		public override string Message
		{
			get
			{
				return MSG;
			}
		}

	}

}